packages/components/datepicker/src/components/osds-datepicker/osds-datepicker.tsx

```diff
@@ -24,9 +24,14 @@ export class OsdsDatepicker implements OdsDatepickerAttribute, OdsDatepickerEven
   @Element() el!: HTMLElement;
 
   @State() hasFocus = false;
-  @State() datepickerInstance: Datepicker | undefined = undefined;
-  @State() hiddenInput: HTMLInputElement | undefined = undefined;
-  @State() datepickerElement: HTMLElement | undefined = undefined;
+
+  protected datepickerInstance: Datepicker | undefined = undefined;
```

Comment:
you can use `private` directly as the controller is not a subclass so he can't access it